assets/src/App.css

```diff
@@ -98,3 +98,16 @@ a.RelatedCustomerConversation--link:hover {
   margin-top: 0px;
   margin-bottom: 4px;
 }
+
+.GettingStartedCode .ant-tabs-tab.ant-tabs-tab {
```

Comment:
this css makes the tabs look a bit too dark i think... i was playing around with it and i think this css might work a bit more nicely with the rest of our design?

```css
.GettingStartedCode .ant-tabs-nav::before {
  border-bottom-color: #d9d9d9;
}

.GettingStartedCode .ant-tabs-nav .ant-tabs-tab {
  border-color: #d9d9d9;
}
```

here's the comparison --

**Before (current code):**
![tabs-before](https://user-images.githubusercontent.com/5264279/110117198-89726f80-7d86-11eb-9d5c-c2e002a84234.gif)

**After (suggested code):**
![tabs-after](https://user-images.githubusercontent.com/5264279/110116833-15d06280-7d86-11eb-8885-198e190479b9.gif)

and here's the default antd design for reference... which is probably too light:
<img width="819" alt="Screen Shot 2021-03-05 at 7 44 47 AM" src="https://user-images.githubusercontent.com/5264279/110117343-b7f04a80-7d86-11eb-8682-c6e85b24858d.png">

@cheeseblubber thoughts?

Reply:
I changed the CSS to your suggestion, lmk if there are other changes

Reply:
@a8t  can you remove this `.GettingStartedCode .ant-tabs-tab.ant-tabs-tab` css? (i think it's no longer necessary)

Reply:
Done, thanks. I missed that.